src/sphinx/extensions/maven_plugin.rst

```diff
@@ -53,6 +53,30 @@ It's bound by default to the ``integration-test`` phase.
 
 .. _maven-advanced-configuration:
 
+Verify
+======
+
+To have Gatling run automatically during the integration-test phase,
+for example: ``mvn verify``, add something like the following to your pom.xml
+
+  <plugin>
+    <groupId>io.gatling</groupId>
+    <artifactId>gatling-maven-plugin</artifactId>
+    <version>MANUALLY_REPLACE_WITH_LATEST_VERSION</version>
+    <configuration>
+      <simulationClass>it.mySimulationIT</simulationClass>
+    </configuration>
+    <executions>
+      <execution>
+        <id>mySimulation</id>
+        <phase>integration-test</phase>
```

Comment:
`integration-test` is the default phase for the `test` goal so it can be omitted.